ng-ncc-app/src/error.html

```diff
@@ -0,0 +1,64 @@
H+VbRZn8a2gFT2CTsDE71uRSFfetZm5NPS1fK8/O4jkhkdILCk1iC178K+wSlpKdjBeaxdaW9yrFx865TERmqN+RYyIiYzXsFvAr6RiLSHKkia5v5OSgV0diImphp3xbet1dYM96KdN52Tk30/CPDxHGgSz5LBBSOuq1FQyCd9hnW/o+77HGHktFJPOQgYmS0lf41xomRwxR33nJDq0lQ3lE0GjgPMh3I22Rh/DKwjdlNwpgGWZ1rkDokCCbuK7Gp8n1xuNeLp8zorsCYW8aMuh8Db2D30diIkIko0ivYFjCsIDXKOyQVkyR9vUyui7nCImUsjL3WRkWl1p5rqnPN3t6OVYGUrzfqqmJNPEdGMojuiKfnVDv/vmPegcW5FFmo9bFtxqKxSfkuUW3RcDbgMSmivewsvK0Da7EmHicFfrshbGIRErUKfxLJCb4M23cd16OXn5IYiLaIKFElcQHnL9aukn8Y2G1V/ll9a5WnvPXJTG9isgqkELaKnP/v0C8JajMRx3NWQS9Tt6gpunjHjH5hvGOGnlXahGNQTTlTg+RiCL8eaV0KPC2oJu8ijtQRMhTzR+7HkjFcxrLdbpk/ZgUDKNvnsSUGEcg0lPElIH94M6hS2vxxcBLmLboGX0Gr1HpGx5E5JvHAX4XsCSmDz1ZxxR5OK8u0UYiEmsqe+zpBS5OeeFXxgtIj8neSnw2IItGiUk9pDkNnF5jLiI3Pe6N11eP6WNBl+xaFPnog8TImTl5VjsC8JN77rTlNdDNWJaUpuj2OnWiHUQIu3kpUQ6+LYru9v5ZB9tz2ZVXDXgDLwFM/kdDRZlRlk+S0pwzMRhcYkiPKMA5txKRDeqH4RLNqPXRr+m5X7jSgVp++HMACvPZkJi+U5wPznHcIin5bCTCFkMM6W169j1f4Jc6/mrw/pMYlWWwAYPW4WG8N/z8DHkm2DVsOiQDYdXkXCoiDC+k92/PvsdX3099SL5Ma6tdHZNvGmFvLaUhtPev4S1NG/BgMjUCfgJY7TXntHrHT185P9bMsuH5+icwERhaSM/XoLkv07GiLRjVNNVFKSPzqsCgPCuwtyz6Cqumqht169OW6oMsjIy4hhf95ElMiwBlYEghvcxAZkLTJV87IqZSsjxSBt1SCR/EF07BQVgIdArgRi9H27Q0bgtiqkTKBvUeW9iGkjLYhDQjDCSkZ+AZ3gX4TSnaD21nZff6qKqLNSCsBljlX0bRJgaP2SJvDlp2Y3ww2owbA3JKKvY981XeK0vFgLxvn5XB9WAkDxehD3w8Jq31Cw1te+KlOWa0x6BUxCTqtiynKtZabPjubwbPKBX21rMlX+/SKit0i/weoy3yazusrOX5ftfpnsJXbl4D/Ka2dVxpItxv4voCnjUBedLDgtMQhIc2gW2ixcLAi4n1So77YwQrIg9GyiiFTeeO3x6qdgB4NNrrEQLppdcwUs/1nIjItIpeUcKfo37nia1z7v5EBCHzvIiw0vxVMUav9ga6EJEnsGbkBUSTSArXPp/bmKaWpmYkLQyUfQJgrdlN3woeSIz8UrnYaDNvjLIStwf2+mejcfY+S88ok20uIihDTir7S8/1KaPHvrZETJU8zkMXBfbd8jmHFb2lVrA8FQIqbMwmjCQrw2OsBPeuY10qkVpt9KaTbyz3+hBCel+M5un92DmlXpL5DGBtIEcLo9+xMIpSL2LSs6YVhouhE3Mly9sDMYAfelNtVNiYseHGPGr9otr16F0ZSI3uQ32+1Tsi9DxLT+fLQpFPVPZ/iciy8LNWuX+Cf+PYUtmuLWVkV731+ejV6jMMs0r+jQW1ZwXZcn4i5DH0tSiQN4S12JhlvKZNwFM9u8D3DCFL79FQN46Rh4R3P1bRgU3FqMC3hues8vNHJyzKoZ2zZODZUmkLqAcEuwVwH6rya6tUoQHvsdfJUyo3s8CHOavooTS5nzd1ul6MTizAG4ZTdLutsZhDxrcLG++HE95faEpmoXuvTbwYegG7M5I+k1OKcEP+j1VDwA2H82o9d1TCzc8GoGgfGcKrJMgLXFbN2/jMt4RCTlkXCk8Vk2WixVPFIuRL3ANvCK9VlI9R05SxWatWalTSbe2zJ/HGLLx64YILGmtcgmi7lvMU/xXAdqFon42Njd7XQzrnZgHtg0cdT6uezRnUboI9KjH5GfI2Jn0kp4Vzjll49TblqsVN6Xv2Nz7XEkbJ6bajCMGLc+4+gFCy5V6I+x7S25ObriIIW+SF3m+e37FtQPZre+GjkoPuIzktPC0MklM7YbCZ0eWTkwpy3lYSzEa9pOdA1jOFbalI70N6BSPtBu2XICwAXBteqWGt32uPa1Rh8vtETiSl8C3GFfLu4wujTfO5rOWoRHGN5s4QNkq41wF2S7Am5flA9sFWoy9Nys0+IVknbEWGz8p8MktHFSc/6zDcYRauICnZW4zOuWv1nnwJKlUv4nYvPu2rwCNtqlr2mzYqJ9fIQ1wWMp8iD7tch3quaVhEusMgQnoH5Ob/DOUGKNzorIRkaghqb0dLYvJqEOtqfsRYLaHJgYVxXQuHboSnA27q/YBu3exy/mPkBYMfVdjjE6GDTH9+okaFeMvfFel3fdBvinE842+l37e7kZdyN9z9MFa5iXVPjHE6ISdT2fm+2x8+RFTCMLiDfZeV686uDRKRB22rIYXK/RAE4Vn+xHIgrfkJgiD29WHbePcd88jnL2smyA3C7a23RR7Lv2XxLEEQRCsIpwBfRCbahDAUj+mVXhJBEPSYWsWaM04QBEGEREzPFmN2XDaCIIh+E1NLr9oiT3rY+j5oxGUjCIIgDPBidZZPj4kgCIIeky8y59yN1cPoMREEQRA+2MK4NRmJiSAIgvDBvfW1QSQmgiAIoq6ndNNEVxMSE0EQBFEVC+QZeBmngiAIgqgEwxqlXyIyb+MqkysuG0EQBFFAhv+umckA/Iu8EXFr3tH/DwDrE4hR8T4SHQAAAABJRU5ErkJggg==" alt="Hackney">
```

Comment:
Why we are using the base64 image instead of just png/svg?
Base64 not ideal: https://csswizardry.com/2017/02/base64-encoding-and-performance/